src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -115,6 +117,20 @@ export default function connectHitsPerPage(renderFn) {
     }
 
     return {
+      getConfiguration() {
+        const defaultValues = items.filter(item => item.default);
+        if (defaultValues.length > 1 && window.console) {
```

Comment:
This can be tested when the widget when the widget is instanciated. At this point, we can safely throw an error (instead of just using a warning in the console).